subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/radio/radio.h

```diff
@@ -5,6 +5,8 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+enum radio_end_evt_delay_state { END_EVT_DELAY_DISABLED, END_EVT_DELAY_ENABLED };
```

Comment:
Yes it could be, but then it would have been better to use some macro instead of true/false. That would have made the code more readable in call locations e.g.:
`sw_switch(SW_SWITCH_RX, SW_SWITCH_TX, phy_rx, flags_rx, phy_tx, flags_tx, END_EVT_DELAY_DISABLED);`
instead of:
`sw_switch(SW_SWITCH_RX, SW_SWITCH_TX, phy_rx, flags_rx, phy_tx, flags_tx, false);`

Is it ok to leave the `enum`? Or do you still think that `boolean` is better here?

Reply:
I will stick to the enum, as the parameter value passed is readable.

Reply:
I'm not going to block this PR on this, but arguing that named macros/enums should be used instead of boolean true/false is a bit odd to me. If it makes to do here, doesn't it make sense to do it everywhere, and avoid using the `bool` type anywhere?

Personally I don't see this specific function as being unique, so if you prefer named defines here for a boolean value (enable/disable, which we have a lot of places in the codebase), do you also prefer that for the other cases we have?

Reply:
Well, for me it should be decided on particular case. E.g. in examples from my comment: https://github.com/zephyrproject-rtos/zephyr/pull/41406#discussion_r774108676 I prefer named constants instead of `true`, `false`. 
When I spot like like the one with `false` then I have to look into function declaration to check what is the argument purpose. `false` does not carry enough information for me. With named constant I have missing information.

Reply:
In this case, we are talking about an enable/disable parameter, which we more often than not have used `bool` for in the code base. 

> When I spot line like the one with false then I have to look into function declaration to check what is the argument purpose. 

But isn't that true for every single parameter that isn't a named enum? For any `uintX_t` you'd also need to check the function declaration to know what it means.

Reply:
> In this case, we are talking about an enable/disable parameter, which we more often than not have used bool for in the code base.

For me that depends on the function, it name and parameters list. e.g.  
```
void start_fsm(bool enable);

// somewhere else in a code 

start_fsm(true);
// or 
start_fsm(false);
```
is quite clear.

If a parameters list is longer then things become more unclear and one needs to look into declaration.

Reply:
As mentioned, I will not block this PR, but I am still in favor using `bool` instead of a named enum for a boolean value.